environment.yml

```
@@ -4,4 +4,4 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python==3.7
+  - python==3.9
```

Might as well bump this to 3.12

```suggestion
  - python==3.12
```